modules/rank-eval/src/main/java/org/elasticsearch/index/rankeval/RankEvalResponse.java

```diff
@@ -122,14 +122,14 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         builder.startObject();
         builder.field("quality_level", evaluationResult);
         builder.startObject("details");
-        for (String key : details.keySet()) {
-            details.get(key).toXContent(builder, params);
+        for (Map.Entry<String, EvalQueryQuality> entry : details.entrySet()) {
```

Comment:
Looks like this doesn't actually need an entry set, but instead the values could be iterated over?